packages/cli/src/index.ts

```diff
@@ -117,12 +122,34 @@ cli.command('deploy', 'deploy the app to the marketplace')
     .option('--dryRun, --dry-run', '[boolean] enable the dry run mode', { default: false })
     .option('--noVerify, --no-verify', '[boolean] disable the linting and typechecking', { default: false })
     .option('--open', '[boolean] open the marketplace app page', { default: false })
+    .option('--appType [appType], --app-type', '[string] specify app type. Overrides manifest values')
     .action(async (options) => {
-        await createDeployment(options.entryPath, options.outDir, {
-            dryRun: options.dryRun,
-            noVerify: options.noVerify,
-            openInBrowser: options.open,
-        });
+        const manifest = reactiveJson<AppManifest>(join(process.cwd(), 'manifest.json'));
+        const appType = options.appType ?? manifest.appType;
```

Comment:
```suggestion
        const appType = options.appType || manifest.appType;
```

Reply:
ah shit sorry, this one was sneaking back in from a merge earlier ...